apis/python/README.md

```diff
@@ -93,3 +93,7 @@ interface TDBFilterSpec {
   [kwarg: string]: any;
 }
 ```
+
+# Information for developers
+
+Please see [https://github.com/single-cell-data/TileDB-SOMA/wiki](https://github.com/single-cell-data/TileDB-SOMA/wiki).
```

Comment:
Micro-nit not worth changing but  I would write it as 

    Please see [the wiki](https://github.com/single-cell-data/TileDB-SOMA/wiki) for additional information.